Gems/PhysX/Code/Tests/EditorTestUtilities.cpp

```diff
@@ -104,6 +120,58 @@ namespace PhysXEditorTests
         return editorEntity;
     }
 
+    EntityPtr CreateCylinderColliderEditorEntity(
+        const AZ::Transform& transform,
+        const AZ::Vector3& positionOffset,
+        const AZ::Quaternion& rotationOffset,
+        float radius,
+        float height,
+        bool dynamic)
+    {
+        EntityPtr editorEntity = CreateInactiveEditorEntity("CylinderEntity");
+        const auto* colliderComponent = editorEntity->CreateComponent<PhysX::EditorColliderComponent>();
+        if (dynamic)
+        {
+            editorEntity->CreateComponent<PhysX::EditorRigidBodyComponent>();
+        }
+        editorEntity->Activate();
+
+        AZ::EntityComponentIdPair idPair(editorEntity->GetId(), colliderComponent->GetId());
+
+        AZ::TransformBus::Event(editorEntity->GetId(), &AZ::TransformBus::Events::SetWorldTM, transform);
+        PhysX::EditorColliderComponentRequestBus::Event(
+            idPair, &PhysX::EditorColliderComponentRequests::SetShapeType, Physics::ShapeType::Cylinder);
+        PhysX::EditorColliderComponentRequestBus::Event(idPair, &PhysX::EditorColliderComponentRequests::SetCylinderRadius, radius);
+        PhysX::EditorColliderComponentRequestBus::Event(idPair, &PhysX::EditorColliderComponentRequests::SetCylinderHeight, height);
+        PhysX::EditorColliderComponentRequestBus::Event(idPair, &PhysX::EditorColliderComponentRequests::SetColliderOffset, positionOffset);
+        PhysX::EditorColliderComponentRequestBus::Event(
+            idPair, &PhysX::EditorColliderComponentRequests::SetColliderRotation, rotationOffset);
+
+        editorEntity->Deactivate();
+        editorEntity->Activate();
```

Comment:
why deactivate for then activate again?

Reply:
If it's not too late it'd be great to add a comment clarifying that.

Reply:
Ah ok yeah, good call, will do